app/src/app/[lng]/[inventory]/preferences/activities/PreferenceCard.tsx

```diff
@@ -0,0 +1,50 @@
+import type { TFunction } from "i18next";
+import { Button, Text, VStack } from "@chakra-ui/react";
+import Image from "next/image";
+
+export function PreferenceCard({
+  onClick,
+  src,
+  title,
+  t,
+}: {
+  onClick: () => void;
+  src: string;
+  t: TFunction;
+  title: string;
+}) {
+  return (
+    <Button
+      onClick={onClick}
+      variant="outline"
+      width={"300px"}
+      height="auto"
+      borderRadius={"md"}
+      borderColor={"border.neutral"}
+      p={4}
+      display="flex"
+      flexDirection="column"
+      alignItems="start"
+    >
+      <VStack spacing={2} align={"start"}>
+        <Image
+          src={src}
+          alt={"imageAlt"}
+          width={32}
+          height={32}
+          objectFit="cover"
+        />
+        <Text
+          fontSize="label.lg"
+          fontWeight="medium"
+          color="gray.700"
+          textTransform="none"
+          whiteSpace="normal"
+          textAlign="left"
```

Comment:
Extract text styles to a variant of the Text component in `app-theme.ts` that is named like the text variant in our Figma design (e.g. in the Terra UI component library by our design team).